DockerHubfile

```diff
@@ -0,0 +1,29 @@
+#### Create Jar 
+FROM gradle:jdk8 as builder 
+ 
+COPY --chown=gradle:gradle . /home/gradle/app 
+WORKDIR /home/gradle/app 
+ 
+RUN mkdir -p api/src/main/resources/META-INF 
+RUN echo "build.version=$(gradle -q printVersion)" > api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.number=${BUILD_NUMBER:=docker}" >> api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.commit=$(git rev-parse HEAD)" >> api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.date=$(date)" >> api/src/main/resources/META-INF/build-info.properties 
+ 
+RUN gradle installDist bootRepackage 
+ 
+#### Actual DockerFile 
+FROM openjdk:8-jre 
+ 
+MAINTAINER "HMCTS <https://github.com/hmcts>" 
+LABEL maintainer = "HMCTS <https://github.com/hmcts>" 
+ 
+WORKDIR /opt/app 
+COPY --from=builder /home/gradle/app/api/build/install/service-auth-provider-api /opt/app . 
+COPY 
+
+HEALTHCHECK --interval=10s --timeout=10s --retries=10 CMD http_proxy="" curl --silent --fail http://localhost:8080/health 
+ 
+EXPOSE 8080
```

Comment:
the app no longer works on 8080 by default afaik

Reply:
https://github.com/hmcts/service-auth-provider-app/blob/master/api/src/main/resources/application.yml#L2